packages/conf/index.d.ts

```diff
@@ -15,7 +15,9 @@ export declare type LinguiConfig = {
     extractBabelOptions: Object;
     fallbackLocale: string;
     format: CatalogFormat;
+    prevFormat: CatalogFormat;
     formatOptions: CatalogFormatOptions;
+    localeDir: string;
```

Comment:
`localeDir` is an obsolete config replaced by `catalogs`. `prevFormat` is obsolete as well.  Why you've put them back? If they're required somewhere, we might fix that code instead.

Reply:
There are some tests using this configuration, how i passed the Partial<LinguiConfig> to mockConfig function, some tests were failing because this type doesnt exists. 

I'll remove this types from the base and i'll extend the type only for this tests cases? or just remove this tests cases that are deprecated?

Reply:
Hmm, good question. `LinguiConfig` type which is used by `Catalog` is a different than `LinguiConfig` in `conf` package: https://github.com/lingui/js-lingui/blob/aff1d092fc2f2848eefb129b931c488e3232fac9/packages/conf/src/index.ts#L22-L36

I've searched the codebase for `localeDir` and I've found it's still being used in babel plugin... Let's merge this one and figure out how to clean the codebase in next PR

Reply:
Perfect for me, it's a pretty easy task (for first constributors) if we document correctly on an independent issue what's the goal

Reply:
I honestly don't know how it's possible that extract plugin works :) I'll take a look when I have some time, but if it works, it's good enough